docs/pages/access-controls/guides/role-templates.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Teleport Role Templates
-description: Mapping SSO and Local Users Traits to Roles with Template.
+description: In this guide, learn how Teleport's role templates provide a way to describe these and other policies.
```

Comment:
What does "these" refer to here? The policies in this guide? I'm not sure what "other" would mean in that case.

Reply:
+1, I don't understand what this is saying